test/bin/bacalhau.sh

```diff
@@ -32,7 +32,7 @@ create_node() {
 
     # Ensure subsequent nodes automatically connect to this requester, and pick
     # a random port for the HTTP API to avoid collisions
-    if test "$TYPE" = "requester"; then
+    if [[ "$TYPE" =~ "requester" ]]; then
```

Comment:
I think this was already fixed via https://github.com/bacalhau-project/bacalhau/pull/3667. So you can rebase your changes on `main`

Reply:
Rebased